src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(String color, double radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.PI * (radius * radius);
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: circle,"
+                + " area: " + Math.round(getArea())
+                + " sq. units, radius: " + radius
+                + " units, color: " + super.getColor());
```

Comment:
For better code readability, it's recommended to store the result of `Math.round(getArea())` in a local variable before using it in the concatenated string.